vcf2fhir/fhir_helper.py

```diff
@@ -597,3 +547,25 @@ def generate_final_json(self):
     def export_fhir_json(self, output_filename):
         with open(output_filename, 'w') as fp:
             json.dump(self.fhir_json, fp, indent=4)
+
+
+def _append(value_to, value_from, list_of_sub_keys, code):
+    value = OrderedDict()
+    key = None
+    is_category = False
+    if code is 0:
+        key = 'category'
+        is_category = True
+    elif code is 1:
+        key = 'code'
+    elif code is 2:
+        key = 'valueCodeableConcept'

```

Comment:
Instead of converting code to key we can directly take `key` is input to function.

Reply:
yes, I tried that but then it was violating the pep 8 rules as it increases the number of characters in one line.